src/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,106 @@
+import React, {FC, FormEvent} from 'react';
+import styled from 'styled-components';
+
+import {Text} from '../Text';
+import {Box} from '../Box';
+
+import {theme} from '../theme';
+
+type TextareaProps = {
+  /** ID, usually used for tests  */
+  id: string;
+  /** className attribute to apply classses from props */
+  className?: string;
+  /** Placeholder */
+  placeholder?: string;
+  /** label displayed above the input  */
+  label?: string;
+  /** used for label - input connection */
+  name?: string;
+  /** input value */
+  value: string;
+  /** error flag */
+  error?: boolean;
+  /** error text message */
+  errorMsg?: string;
+  /** Allow user to resize the textarea vertically and horizontally or not */
+  resize: 'none' | 'both';
```

Comment:
shall we set `resize` as an optional prop that defaults to ***none***?

the number of use cases of users adjusting the size in practice is extremely low (personal assumption)